packages/Webkul/Admin/src/Resources/views/configuration/field-type.blade.php

```diff
@@ -37,7 +37,7 @@
 
         @elseif ($field['type'] == 'number')
 
-            <input type="number" min="0" v-validate="'{{ $validations }}'" class="control" id="{{ $name }}" name="{{ $name }}" value="{{ old($nameKey) ?: core()->getConfigData($nameKey, $channel, $locale) }}" data-vv-as="&quot;{{ trans($field['title']) }}&quot;">
+            <input type="number" min="1" v-validate="'{{ $validations }}'" class="control" id="{{ $name }}" name="{{ $name }}" value="{{ old($nameKey) ?: core()->getConfigData($nameKey, $channel, $locale) }}" data-vv-as="&quot;{{ trans($field['title']) }}&quot;">
```

Comment:
This will impact all field types with numbers on the configuration page. Some of the fields may have zero also.